packages/augur-ui/src/services/augursdk.ts

```diff
@@ -104,9 +114,30 @@ export class SDK {
 
     window.AugurSDK = this.sdk;
 
+    this.addErrorHandler(meshBrowser, this.meshConfig);
+    window.AugurSDK.zeroX.setMesh(meshBrowser);
+    window.AugurSDK.zeroX.startAsync();
+
     return this.sdk;
   }
 
+  addErrorHandler(mesh: browserMesh, meshConfig: Config) {
```

Comment:
No need to pass in `meshConfig` here, it exists on `this`

Also, this code does the same logic as connecting in the first place.

Would suggest something like
```typescript
connectToBrowserMesh() { 
   const mesh = new Mesh(this.meshConfig);
   mesh.onError((err) => {
       if(err.message === MESH_TIMEOUT_ERROR) {
          this.connectToBrowserMesh();
       }
   });

   this.sdk.zeroX.mesh = mesh;
   mesh.startAsync();
}